internal/pkg/plugin/githubactions/workflow.go

```diff
@@ -12,6 +11,7 @@ const (
 	defaultCommitMessage = "builder by DevStream"
 	BuilderYmlPr         = "pr-builder.yml"
 	BuilderYmlMaster     = "master-builder.yml"
+	BuilderYmlPrMaster   = "pr-master-builder.yml"
```

Comment:
Maybe we need a better name, with "-devstream.yaml" as suffix.

Reply:
Or maybe we don't need the suffix; only a better name.

Reply:
done